src/components/advertisementFeature/logo.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Item } from 'item';
+import { TagType } from '@guardian/content-api-models/v1/tagType';
+import { css } from '@emotion/core';
+import { textSans } from '@guardian/src-foundations/typography';
+import { darkModeCss } from 'styles';
+import { remSpace, neutral, text } from '@guardian/src-foundations';
+
+interface Props {
+    item: Item;
+}
+
+const styles = css`
+    display: flex;
+    justify-content: space-between;
+    ${textSans.medium()}
+    color: ${text.supporting};
+
+    ${darkModeCss`
+        color: ${neutral[60]};
+        img {
+            padding: ${remSpace[2]};
+            background: ${neutral[86]};
+        }
+    `}
+`;
+
+const Logo = ({ item }: Props): JSX.Element => {
+    const sponsorship = item.tags
+        .find(tag => tag.type === TagType.PAID_CONTENT)?.activeSponsorships?.pop();
+    const logo = sponsorship?.sponsorLogo;
+    const link = sponsorship?.sponsorLink;
+    const alt = sponsorship?.sponsorName ?? "";
```

Comment:
We could ask the parser to do this work for us. To do this we might include a logo field in the `AdvertisementFeature` item, similar to how we only include `starRating` for `Review` articles. That would pull some of the responsibility for handling the CAPI response out of this component.